conf/index.html

```diff
@@ -1,87 +1,116 @@
-<!DOCTYPE html>
-<html>
+<html  xmlns="http://www.w3.org/1999/xhtml"
```

Comment:
Yes, it will be separated in upcoming fix when I move it under widgets for the dashboard. Right now it's still evolving. Good suggestion.